src/md/contributing.md

```diff
@@ -41,7 +42,7 @@ If you are new to git, this [tutorial](https://try.github.io/) may be helpful.
 
 The ASF Apex git repositories have mirror repositories on github which are used to review pull requests and provide a second remote endpoint for the codebase.
 
-1. Fork the ASF github mirror: https://github.com/apache/apex-core (or https://github.com/apache/apex-malhar or https://github.com/apache/apex-site) 
+1. Fork the ASF repository: https://github.com/apache/apex-core (or https://github.com/apache/apex-malhar or https://github.com/apache/apex-site) 
```

Comment:
github repos are still mirrors of gitbox repos.

Reply:
For all practical purposes, we only work with github.

Reply:
Yes, should the line above be changed to remove references to mirror as well? Should it say "are used to review and merge pull requests"? Fork the github [Apex Core](https://github.com/apache/apex-core) ... to your github account?

Reply:
Clickable link that goes to github mirror is better than the link spelled out because when users click on the link and it goes to the github page, they see many options to fork clone etc.